hammerdb/azuredeploy.json

```diff
@@ -248,6 +249,20 @@
                             "priority": 101,
                             "direction": "Inbound"
                         }
+                    },
+                    {
+                        "name": "customSshPort",
+                        "properties": {
+                            "description": "custom ssh port",
+                            "protocol": "Tcp",
+                            "sourcePortRange": "*",
+                            "destinationPortRange": "[variables('customSshPort')]",
+                            "sourceAddressPrefix": "*",
+                            "destinationAddressPrefix": "*",
+                            "access": "Allow",
+                            "priority": 102,
```

Comment:
what happens if there is already a rule with `"priority": 102,`? Or, could that happen?

Reply:
there is already a rule with 102 :)

It gets appended after all the existing rules. So 100, 101, 102, and 103 will be reserved. and these rules will have priority of 104, 105...